docs/operation/cluster_and_service_configuration.rst

```diff
@@ -394,6 +395,11 @@ In order to add Alert rules, a new ``PrometheusRule`` manifest must be created.
          labels:
            severity: warning
 
+.. note::
+
+   All the labels in the example above are mandatory for Prometheus
+   to take the new rules into account.
```

Comment:
That seems like an issue, no? The app deploys the rule, so `app.kubernetes.io/name` and `app` must be related to the app *creating* the rule, totally unrelated to `prometheus-operator`. Similarly, `release` should not be part of any (required) set of labels, and should not be `prometheus-release`.

In essence, we need to *change* the set of labels our `prometheus-operator` uses to filter out rules (and other objects) it should take into account, e.g., turn the selector into `metalk8s.scality.com/monitor: "true"` or something along those lines.

Reply:
Can easily be overridden using `prometheus.prometheusSpec.ruleSelector` in the `charts/kube-prometheus-stack.yaml` file.

Reply:
Indeed, that's what we recommend in #2199
This PR was more about making sure our docs are accurate about the current state of the product (although it's debatable whether it's a useful change :stuck_out_tongue_winking_eye:)

Reply:
Oneline fix, does not cost much, at least now it's accurate even if it's ugly (which was already the case).